### ISSUE TITLE
Nickname/description for added multiplayer servers

### ISSUE THREAD

**Issue by runelaenen** [OpenRCT2/OpenRCT2#2032]
I have a few of them saved and I'd like to tell my friends' servers apart. 
It's hard to memorize IPs!

Topic: https://openrct.net/forum/topic/587/last/nickname-for-added-multiplayer-servers#addReply

**Comment by IntelOrca** [comment]
Ideally it should pulse the server for a name (perhaps using UDP) and then cache that (update it if possible) when opening the server list window.

**Comment by Patrik356b** [comment]
Yes, a common update interval is 1~5 seconds and that would be appropriate

**Comment by pizza2004** [comment]
I like the idea of the server itself having a description, but I think the user should be able to name servers still.  I'm thinking of Minecraft, where it's convenient to name the server "Friend's Server" with friend being the name of the specific person, but I still get a description from the server of itself, or at the least can.  Otherwise it could become more difficult to tell which of your saved servers are which.

Perhaps if we get to a point where we have a list of open servers then we can just grab the name from the server, but I'd still prefer to be able to give my favorites custom names for my own benefit.

**Comment by IntelOrca** [comment]
@pizza2004 OK

**Comment by Nubbie** [comment]
The servers you've saved now shows the name that it last used, isn't this obsolete now as IP's isn't shown anymore?
